2024-06-15:14:24:36,965 ERROR    [carbon_service.py:43] carbon_service - create_carbon_score - TypeError raised 
2024-06-20:17:03:04,964 ERROR    [api_service.py:66] TypeError raised conversion from NoneType to Decimal is not supported
2024-06-20:17:03:04,966 ERROR    [api_service.py:70] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:03:04,967 ERROR    [api_service.py:118] TypeError raised leg is not a dict
2024-06-20:17:03:04,967 ERROR    [api_service.py:122] Exception raised invalid literal for int() with base 10: 'test'
2024-06-20:17:03:04,968 ERROR    [api_service.py:236] TypeError raised source is not a string
2024-06-20:17:03:04,968 ERROR    [api_service.py:240] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:03:04,969 ERROR    [api_service.py:185] TypeError raised weight_unit is not a string
2024-06-20:17:03:04,969 ERROR    [api_service.py:189] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:03:04,970 ERROR    [api_service.py:89] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:03:04,970 ERROR    [api_service.py:145] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:03:04,971 ERROR    [api_service.py:259] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:03:04,971 ERROR    [api_service.py:208] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:03:05,067 ERROR    [carbon_service.py:67] TypeError raised 
2024-06-20:17:03:05,069 ERROR    [carbon_service.py:43] TypeError raised 
2024-06-20:17:03:05,081 ERROR    [database_interface.py:123] Exception raised (1065, 'Query was empty')
2024-06-20:17:03:05,091 ERROR    [database_interface.py:70] Exception raised (1065, 'Query was empty')
2024-06-20:17:03:05,102 ERROR    [database_interface.py:96] ValueError raised 
2024-06-20:17:03:05,112 ERROR    [database_interface.py:37] ValueError raised 
2024-06-20:17:03:05,112 ERROR    [database_interface.py:37] ValueError raised 
2024-06-20:17:03:05,114 ERROR    [database_interface.py:153] ValueError raised 
2024-06-20:17:03:05,124 ERROR    [shipping_service.py:54] TypeError raised 
2024-06-20:17:03:05,398 ERROR    [estimates_service.py:86] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:03:05,452 ERROR    [estimates_service.py:57] Exception raised 
2024-06-20:17:03:05,454 ERROR    [fuel_combustion_service.py:59] TypeError raised 
2024-06-20:17:03:27,810 ERROR    [api_service.py:66] TypeError raised conversion from NoneType to Decimal is not supported
2024-06-20:17:03:27,810 ERROR    [api_service.py:70] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:03:27,812 ERROR    [api_service.py:118] TypeError raised leg is not a dict
2024-06-20:17:03:27,812 ERROR    [api_service.py:122] Exception raised invalid literal for int() with base 10: 'test'
2024-06-20:17:03:27,813 ERROR    [api_service.py:236] TypeError raised source is not a string
2024-06-20:17:03:27,813 ERROR    [api_service.py:240] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:03:27,815 ERROR    [api_service.py:185] TypeError raised weight_unit is not a string
2024-06-20:17:03:27,815 ERROR    [api_service.py:189] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:03:27,815 ERROR    [api_service.py:89] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:03:27,816 ERROR    [api_service.py:145] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:03:27,817 ERROR    [api_service.py:259] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:03:27,817 ERROR    [api_service.py:208] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:03:27,863 ERROR    [api_service.py:66] TypeError raised conversion from NoneType to Decimal is not supported
2024-06-20:17:03:27,863 WARNING  [log.py:241] Bad Request: /api/create/electricity/
2024-06-20:17:03:27,921 ERROR    [carbon_service.py:67] TypeError raised 
2024-06-20:17:03:27,926 ERROR    [carbon_service.py:43] TypeError raised 
2024-06-20:17:03:27,936 ERROR    [database_interface.py:123] Exception raised (1065, 'Query was empty')
2024-06-20:17:03:27,946 ERROR    [database_interface.py:70] Exception raised (1065, 'Query was empty')
2024-06-20:17:03:27,954 ERROR    [database_interface.py:96] ValueError raised 
2024-06-20:17:03:27,965 ERROR    [database_interface.py:37] ValueError raised 
2024-06-20:17:03:27,966 ERROR    [database_interface.py:37] ValueError raised 
2024-06-20:17:03:27,968 ERROR    [database_interface.py:153] ValueError raised 
2024-06-20:17:03:27,982 ERROR    [shipping_service.py:54] TypeError raised 
2024-06-20:17:03:28,004 ERROR    [estimates_service.py:86] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:03:28,056 ERROR    [estimates_service.py:57] Exception raised 
2024-06-20:17:03:28,057 ERROR    [fuel_combustion_service.py:59] TypeError raised 
2024-06-20:17:03:37,524 ERROR    [api_service.py:66] TypeError raised conversion from NoneType to Decimal is not supported
2024-06-20:17:03:37,524 ERROR    [api_service.py:70] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:03:37,526 ERROR    [api_service.py:118] TypeError raised leg is not a dict
2024-06-20:17:03:37,526 ERROR    [api_service.py:122] Exception raised invalid literal for int() with base 10: 'test'
2024-06-20:17:03:37,527 ERROR    [api_service.py:236] TypeError raised source is not a string
2024-06-20:17:03:37,527 ERROR    [api_service.py:240] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:03:37,528 ERROR    [api_service.py:185] TypeError raised weight_unit is not a string
2024-06-20:17:03:37,528 ERROR    [api_service.py:189] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:03:37,529 ERROR    [api_service.py:89] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:03:37,529 ERROR    [api_service.py:145] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:03:37,530 ERROR    [api_service.py:259] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:03:37,530 ERROR    [api_service.py:208] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:03:37,564 ERROR    [api_service.py:66] TypeError raised conversion from NoneType to Decimal is not supported
2024-06-20:17:03:37,566 WARNING  [log.py:241] Bad Request: /api/create/electricity/
2024-06-20:17:03:37,612 ERROR    [carbon_service.py:67] TypeError raised 
2024-06-20:17:03:37,614 ERROR    [carbon_service.py:43] TypeError raised 
2024-06-20:17:03:37,625 ERROR    [database_interface.py:123] Exception raised (1065, 'Query was empty')
2024-06-20:17:03:37,636 ERROR    [database_interface.py:70] Exception raised (1065, 'Query was empty')
2024-06-20:17:03:37,646 ERROR    [database_interface.py:96] ValueError raised 
2024-06-20:17:03:37,657 ERROR    [database_interface.py:37] ValueError raised 
2024-06-20:17:03:37,657 ERROR    [database_interface.py:37] ValueError raised 
2024-06-20:17:03:37,659 ERROR    [database_interface.py:153] ValueError raised 
2024-06-20:17:03:37,669 ERROR    [shipping_service.py:54] TypeError raised 
2024-06-20:17:03:37,688 ERROR    [estimates_service.py:86] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:03:37,721 ERROR    [estimates_service.py:57] Exception raised 
2024-06-20:17:03:37,723 ERROR    [fuel_combustion_service.py:59] TypeError raised 
2024-06-20:17:04:14,360 ERROR    [api_service.py:66] TypeError raised conversion from NoneType to Decimal is not supported
2024-06-20:17:04:14,361 ERROR    [api_service.py:70] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:04:14,362 ERROR    [api_service.py:118] TypeError raised leg is not a dict
2024-06-20:17:04:14,362 ERROR    [api_service.py:122] Exception raised invalid literal for int() with base 10: 'test'
2024-06-20:17:04:14,363 ERROR    [api_service.py:236] TypeError raised source is not a string
2024-06-20:17:04:14,363 ERROR    [api_service.py:240] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:04:14,364 ERROR    [api_service.py:185] TypeError raised weight_unit is not a string
2024-06-20:17:04:14,364 ERROR    [api_service.py:189] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:04:14,364 ERROR    [api_service.py:89] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:04:14,365 ERROR    [api_service.py:145] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:04:14,365 ERROR    [api_service.py:259] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:04:14,366 ERROR    [api_service.py:208] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:04:14,395 ERROR    [api_service.py:66] TypeError raised conversion from NoneType to Decimal is not supported
2024-06-20:17:04:14,396 WARNING  [log.py:241] Bad Request: /api/create/electricity/
2024-06-20:17:04:14,410 ERROR    [api_service.py:66] TypeError raised conversion from NoneType to Decimal is not supported
2024-06-20:17:04:14,411 WARNING  [log.py:241] Bad Request: /api/create/electricity/
2024-06-20:17:04:14,456 ERROR    [carbon_service.py:67] TypeError raised 
2024-06-20:17:04:14,457 ERROR    [carbon_service.py:43] TypeError raised 
2024-06-20:17:04:14,465 ERROR    [database_interface.py:123] Exception raised (1065, 'Query was empty')
2024-06-20:17:04:14,475 ERROR    [database_interface.py:70] Exception raised (1065, 'Query was empty')
2024-06-20:17:04:14,484 ERROR    [database_interface.py:96] ValueError raised 
2024-06-20:17:04:14,492 ERROR    [database_interface.py:37] ValueError raised 
2024-06-20:17:04:14,493 ERROR    [database_interface.py:37] ValueError raised 
2024-06-20:17:04:14,494 ERROR    [database_interface.py:153] ValueError raised 
2024-06-20:17:04:14,503 ERROR    [shipping_service.py:54] TypeError raised 
2024-06-20:17:04:14,524 ERROR    [estimates_service.py:86] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:04:14,554 ERROR    [estimates_service.py:57] Exception raised 
2024-06-20:17:04:14,554 ERROR    [fuel_combustion_service.py:59] TypeError raised 
2024-06-20:17:04:52,608 ERROR    [api_service.py:66] TypeError raised conversion from NoneType to Decimal is not supported
2024-06-20:17:04:52,609 ERROR    [api_service.py:70] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:04:52,610 ERROR    [api_service.py:118] TypeError raised leg is not a dict
2024-06-20:17:04:52,610 ERROR    [api_service.py:122] Exception raised invalid literal for int() with base 10: 'test'
2024-06-20:17:04:52,611 ERROR    [api_service.py:236] TypeError raised source is not a string
2024-06-20:17:04:52,611 ERROR    [api_service.py:240] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:04:52,612 ERROR    [api_service.py:185] TypeError raised weight_unit is not a string
2024-06-20:17:04:52,612 ERROR    [api_service.py:189] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:04:52,613 ERROR    [api_service.py:89] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:04:52,613 ERROR    [api_service.py:145] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:04:52,614 ERROR    [api_service.py:259] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:04:52,614 ERROR    [api_service.py:208] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:04:52,644 ERROR    [api_service.py:66] TypeError raised conversion from NoneType to Decimal is not supported
2024-06-20:17:04:52,645 WARNING  [log.py:241] Bad Request: /api/create/electricity/
2024-06-20:17:04:52,655 ERROR    [api_service.py:66] TypeError raised conversion from NoneType to Decimal is not supported
2024-06-20:17:04:52,655 WARNING  [log.py:241] Bad Request: /api/create/electricity/
2024-06-20:17:04:52,707 ERROR    [carbon_service.py:67] TypeError raised 
2024-06-20:17:04:52,708 ERROR    [carbon_service.py:43] TypeError raised 
2024-06-20:17:04:52,719 ERROR    [database_interface.py:123] Exception raised (1065, 'Query was empty')
2024-06-20:17:04:52,731 ERROR    [database_interface.py:70] Exception raised (1065, 'Query was empty')
2024-06-20:17:04:52,741 ERROR    [database_interface.py:96] ValueError raised 
2024-06-20:17:04:52,748 ERROR    [database_interface.py:37] ValueError raised 
2024-06-20:17:04:52,749 ERROR    [database_interface.py:37] ValueError raised 
2024-06-20:17:04:52,750 ERROR    [database_interface.py:153] ValueError raised 
2024-06-20:17:04:52,756 ERROR    [shipping_service.py:54] TypeError raised 
2024-06-20:17:04:52,771 ERROR    [estimates_service.py:86] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:04:52,797 ERROR    [estimates_service.py:57] Exception raised 
2024-06-20:17:04:52,798 ERROR    [fuel_combustion_service.py:59] TypeError raised 
2024-06-20:17:05:17,547 ERROR    [api_service.py:66] TypeError raised conversion from NoneType to Decimal is not supported
2024-06-20:17:05:17,547 ERROR    [api_service.py:70] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:05:17,548 ERROR    [api_service.py:118] TypeError raised leg is not a dict
2024-06-20:17:05:17,549 ERROR    [api_service.py:122] Exception raised invalid literal for int() with base 10: 'test'
2024-06-20:17:05:17,549 ERROR    [api_service.py:236] TypeError raised source is not a string
2024-06-20:17:05:17,549 ERROR    [api_service.py:240] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:05:17,550 ERROR    [api_service.py:185] TypeError raised weight_unit is not a string
2024-06-20:17:05:17,550 ERROR    [api_service.py:189] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:05:17,551 ERROR    [api_service.py:89] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:05:17,551 ERROR    [api_service.py:145] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:05:17,552 ERROR    [api_service.py:259] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:05:17,552 ERROR    [api_service.py:208] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:05:17,587 ERROR    [api_service.py:66] TypeError raised conversion from NoneType to Decimal is not supported
2024-06-20:17:05:17,587 WARNING  [log.py:241] Bad Request: /api/create/electricity/
2024-06-20:17:05:17,605 ERROR    [api_service.py:118] TypeError raised unit is not a string
2024-06-20:17:05:17,606 WARNING  [log.py:241] Bad Request: /api/create/flight/
2024-06-20:17:05:17,630 ERROR    [carbon_service.py:67] TypeError raised 
2024-06-20:17:05:17,631 ERROR    [carbon_service.py:43] TypeError raised 
2024-06-20:17:05:17,639 ERROR    [database_interface.py:123] Exception raised (1065, 'Query was empty')
2024-06-20:17:05:17,648 ERROR    [database_interface.py:70] Exception raised (1065, 'Query was empty')
2024-06-20:17:05:17,663 ERROR    [database_interface.py:96] ValueError raised 
2024-06-20:17:05:17,671 ERROR    [database_interface.py:37] ValueError raised 
2024-06-20:17:05:17,671 ERROR    [database_interface.py:37] ValueError raised 
2024-06-20:17:05:17,672 ERROR    [database_interface.py:153] ValueError raised 
2024-06-20:17:05:17,677 ERROR    [shipping_service.py:54] TypeError raised 
2024-06-20:17:05:17,691 ERROR    [estimates_service.py:86] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:05:17,714 ERROR    [estimates_service.py:57] Exception raised 
2024-06-20:17:05:17,715 ERROR    [fuel_combustion_service.py:59] TypeError raised 
2024-06-20:17:07:36,810 ERROR    [api_service.py:66] TypeError raised conversion from NoneType to Decimal is not supported
2024-06-20:17:07:36,810 ERROR    [api_service.py:70] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:07:36,811 ERROR    [api_service.py:118] TypeError raised leg is not a dict
2024-06-20:17:07:36,811 ERROR    [api_service.py:122] Exception raised invalid literal for int() with base 10: 'test'
2024-06-20:17:07:36,812 ERROR    [api_service.py:236] TypeError raised source is not a string
2024-06-20:17:07:36,812 ERROR    [api_service.py:240] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:07:36,813 ERROR    [api_service.py:185] TypeError raised weight_unit is not a string
2024-06-20:17:07:36,813 ERROR    [api_service.py:189] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:07:36,813 ERROR    [api_service.py:89] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:07:36,813 ERROR    [api_service.py:145] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:07:36,814 ERROR    [api_service.py:259] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:07:36,814 ERROR    [api_service.py:208] Exception raised 'str' object has no attribute 'get'
2024-06-20:17:07:36,855 ERROR    [api_service.py:66] TypeError raised conversion from NoneType to Decimal is not supported
2024-06-20:17:07:36,855 WARNING  [log.py:241] Bad Request: /api/create/electricity/
2024-06-20:17:07:36,874 ERROR    [api_service.py:118] TypeError raised unit is not a string
2024-06-20:17:07:36,875 WARNING  [log.py:241] Bad Request: /api/create/flight/
2024-06-20:17:07:36,895 ERROR    [api_service.py:236] TypeError raised source is not a string
2024-06-20:17:07:36,895 WARNING  [log.py:241] Bad Request: /api/create/fuel/
2024-06-20:17:07:36,897 ERROR    [api.py:838] TypeError raised 
2024-06-20:17:07:36,898 WARNING  [log.py:241] Bad Request: /api/get/score/
2024-06-20:17:07:36,916 ERROR    [api_service.py:185] TypeError raised conversion from NoneType to Decimal is not supported
2024-06-20:17:07:36,916 WARNING  [log.py:241] Bad Request: /api/create/shipping/
2024-06-20:17:07:36,926 ERROR    [carbon_service.py:67] TypeError raised 
2024-06-20:17:07:36,927 ERROR    [carbon_service.py:43] TypeError raised 
2024-06-20:17:07:36,937 ERROR    [database_interface.py:123] Exception raised (1065, 'Query was empty')
2024-06-20:17:07:36,945 ERROR    [database_interface.py:70] Exception raised (1065, 'Query was empty')
2024-06-20:17:07:36,956 ERROR    [database_interface.py:96] ValueError raised 
2024-06-20:17:07:36,963 ERROR    [database_interface.py:37] ValueError raised 
2024-06-20:17:07:36,963 ERROR    [database_interface.py:37] ValueError raised 
2024-06-20:17:07:36,964 ERROR    [database_interface.py:153] ValueError raised 
2024-06-20:17:07:36,971 ERROR    [shipping_service.py:54] TypeError raised 
2024-06-20:17:07:36,986 ERROR    [estimates_service.py:86] Exception raised [<class 'decimal.ConversionSyntax'>]
2024-06-20:17:07:37,014 ERROR    [estimates_service.py:57] Exception raised 
2024-06-20:17:07:37,015 ERROR    [fuel_combustion_service.py:59] TypeError raised 
